Source/Particles/Pusher/PushSelector.H

```diff
@@ -128,18 +121,16 @@ void doParticlePush(const GetParticlePosition& GetPosition,
         UpdatePosition(x, y, z, ux, uy, uz, dt );
         SetPosition(i, x, y, z);
     } else if (pusher_algo == ParticlePusherAlgo::HigueraCary) {
-        amrex::ParticleReal qp = q;
-        if (ion_lev){ qp *= ion_lev; }
         UpdateMomentumHigueraCary( ux, uy, uz,
                                    Ex, Ey, Ez, Bx,
                                    By, Bz, qp, m, dt);
         amrex::ParticleReal x, y, z;
         GetPosition(i, x, y, z);
         UpdatePosition(x, y, z, ux, uy, uz, dt );
         SetPosition(i, x, y, z);
-    } else {
-        amrex::Abort("Unknown particle pusher");
-    }
+    } //else {
+//        amrex::Abort("Unknown particle pusher");
+//    }
```

Comment:
```suggestion
    }
#ifdef AMREX_DEBUG
    else {
        amrex::Abort("Unknown particle pusher");
    }
#endif
```

Reply:
Yes, I did mean to comment out or even remove the else part, because it could affect performance and it's not really necessary.  The runtime parameter is converted from a string to int.  If there is a typo in inputs, it would have be caught in WarpX::ReadParameters.  Admittedly, it's still possible that something could go wrong.  But it's very unlikely.  If we really want to make sure there is no unknown type, we could turn the `int` into a strong type like `enum struct`.